### ISSUE TITLE
Problem with cover rollershuter

### ISSUE THREAD

**Issue by gonzalezcalleja** [home-assistant/homebridge-homeassistant#62]
Hi,

I have problems with last version of home bridge-homeassistant and rollershuters (Fibaro FGRM-222), when I run homebridge I get: 

```
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing HomeAssistant platform...
Booting Platform
[10/29/2016, 3:19:12 AM] [HomeAssistant] Fetching HomeAssistant devices.
Plugin - DidFinishLaunching
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'PWS_temp_f'...
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'Detectores de movimiento en camaras interiores para Alarma'...
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'Dark Sky Apparent Temperature'...
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'Luz por movimiento en Planta Primera'...
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'Acuario'...
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'Detectores de sonido en camaras interiores para Alarma'...
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'Luz Planta Baja'...
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'PWS_dewpoint_c'...
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'PWS_heat_index_f'...
[10/29/2016, 3:19:12 AM] [HomeAssistant] Initializing platform accessory 'Persiana Oeste'...
/root/.nvm/versions/node/v6.7.0/lib/node_modules/homebridge-homeassistant/accessories/cover.js:133
      .on("get", this.stateCharacteristicGetFunction.bind(this));
      ^

TypeError: Cannot read property 'on' of undefined
    at Object.getServices (/root/.nvm/versions/node/v6.7.0/lib/node_modules/homebridge-homeassistant/accessories/cover.js:133:7)
    at Server._createAccessory (/root/.nvm/versions/node/v6.7.0/lib/node_modules/homebridge/lib/server.js:375:36)
    at Server.<anonymous> (/root/.nvm/versions/node/v6.7.0/lib/node_modules/homebridge/lib/server.js:361:32)
    at /root/.nvm/versions/node/v6.7.0/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/util/once.js:15:19
    at /root/.nvm/versions/node/v6.7.0/lib/node_modules/homebridge-homeassistant/index.js:205:7
    at Request.onResponse [as _callback] (/root/.nvm/versions/node/v6.7.0/lib/node_modules/homebridge-homeassistant/index.js:111:7)
    at Request.self.callback (/root/.nvm/versions/node/v6.7.0/lib/node_modules/homebridge-homeassistant/node_modules/request/request.js:186:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/root/.nvm/versions/node/v6.7.0/lib/node_modules/homebridge-homeassistant/node_modules/request/request.js:1060:10)

```

This is part of my HA config:

```python
cover.salon1_level_5_0:
  friendly_name: Persiana Este
  homebridge_cover_type: rollershutter
  
cover.salon2_level_6_0:
  friendly_name: Persiana Oeste
  homebridge_cover_type: roller shutter
```

What am i doing wrong?

**Comment by robbiet480** [comment]
@gonzalezcalleja There was a fix pushed mere moments ago, please pull the latest (`npm install -g home-assistant/homebridge-homeassistant`) and try again. 

Just for @oriolism and @jsg4's benefit, this is the bug I mentioned I fixed in #59.

**Comment by gonzalezcalleja** [comment]
Now is working after update, thanks!
